### PR TITLE
Dịch "batch-norm_vn" - Phần 6

### LINKED ISSUE #1384: Dịch "batch-norm_vn" - Phần 6
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_convolutional-modern/batch-norm_vn.md#L305-L405

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by tranyenthy**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #1384 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

**Comment by duythanhvn** [comment]
@tranyenthy Những phần WIP này chừng nào có thể **ready in review** cô gái ơi?

**Comment by tranyenthy** [comment]
> 
> 
> @tranyenthy Những phần WIP này chừng nào có thể **ready in review** cô gái ơi?

Mình sẽ lên ngay. Để bị nhắc rồi ngại quá... :'(

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 1]
```diff
@@ -314,7 +314,13 @@ Also note that for the sake of convenience we did not worry about automatically
 Do not worry, the Gluon `BatchNorm` layer will care of this for us.
 -->
 
-*dịch đoạn phía trên*
+Ta tạm để các chi tiết thuật toán sang một bên, chú ý đến design pattern bên dưới tầng lập trình.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:317]:
```suggestion
Ta tạm để các chi tiết thuật toán sang một bên mà tập trung vào các khuôn mẫu thiết kế nền tảng cho việc lập trình. 
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 2]
```diff
@@ -314,7 +314,13 @@ Also note that for the sake of convenience we did not worry about automatically
...
+Thông thường, ta định nghĩa phần toán trong một hàm riêng biệt, giả sử `batch_norm`.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:318]:
```suggestion
Thông thường, ta định nghĩa phần toán trong một hàm riêng biệt, chẳng hạn như `batch_norm`.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 3]
```diff
@@ -314,7 +314,13 @@ Also note that for the sake of convenience we did not worry about automatically
...
+Sau đó, ta tích hợp chức năng này vào một tầng tùy chỉnh, có mã chủ yếu giải quyết các vấn đề về sổ sách chứng từ,
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:319]:
```suggestion
Sau đó, ta tích hợp chức năng này vào một tầng tùy chỉnh, với mã nguồn chủ yếu để giải quyết các vấn đề sổ sách chứng từ,
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 4]
```diff
@@ -314,7 +314,13 @@ Also note that for the sake of convenience we did not worry about automatically
...
+chẳng hạn như di chuyển dữ liệu đến thiết bị phù hợp ngữ cảnh, phân bổ và khởi tạo bất kỳ biến nào được yêu cầu, theo dõi mức trung bình đang chạy (ở đây là trung bình và phương sai), v.v.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:320]:
```suggestion
chẳng hạn như di chuyển dữ liệu đến thiết bị phù hợp ngữ cảnh, phân bổ và khởi tạo bất kỳ biến nào được yêu cầu, theo dõi các giá trị trung bình động (của trung bình và phương sai trong trường hợp này), v.v.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 5]
```diff
@@ -314,7 +314,13 @@ Also note that for the sake of convenience we did not worry about automatically
...
+Pattern này cho phép tách toán hoàn toàn ra khỏi đoạn mã soạn sẵn.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:321]:
```suggestion
Khuôn mẫu này cho phép ta tách hoàn toàn các phép tính toán ra khỏi đoạn mã phụ trợ.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 6]
```diff
@@ -314,7 +314,13 @@ Also note that for the sake of convenience we did not worry about automatically
...
+Cũng lưu ý rằng để thuận tiện, ta không lo lắng về việc tự động suy ra kích thước đầu vào ở đây, do đó chúng ta cần chỉ định số lượng đặc trưng xuyên suốt.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:322]:
```suggestion
Cũng lưu ý rằng để thuận tiện, ta không cần lo về việc tự động suy ra kích thước đầu vào ở đây, do đó chúng ta cũng không cần chỉ định số lượng đặc trưng xuyên suốt.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 7]
```diff
@@ -384,7 +391,9 @@ This code is virtually identical to that when we first trained LeNet (:numref:`s
 The main difference is the considerably larger learning rate.
 -->
 
-*dịch đoạn phía trên*
+Như trước đây, ta sẽ đào tạo mạng trên bộ dữ liệu Fashion-MNIST.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:394]:
```suggestion
Như trước đây, ta sẽ huấn luyện trên bộ dữ liệu Fashion-MNIST.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 8]
```diff
@@ -384,7 +391,9 @@ This code is virtually identical to that when we first trained LeNet (:numref:`s
...
+Đoạn mã này gần giống với mã này khi chúng tôi lần đầu tiên đào tạo LeNet (: numref: `sec_lenet`).
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:395]:
```suggestion
Đoạn mã này gần như là tương tự với khi chúng ta lần đầu huấn luyên LeNet (: numref: `sec_lenet`).
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 9]
```diff
@@ -396,7 +405,7 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
 Let us have a look at the scale parameter `gamma` and the shift parameter `beta` learned from the first batch normalization layer.
 -->
 
-*dịch đoạn phía trên*
+Chúng ta hãy xem tham số quy mô `gamma` và tham số shift` beta` đã học được từ tầng chuẩn hóa theo batch đầu tiên.
```
Comment (lkhphuc) [inline chapter_convolutional-modern/batch-norm_vn.md:408]:
```suggestion
Chúng ta hãy xem tham số tỷ lệ `gamma` và tham số dịch chuyển ` beta` đã học được tại tầng chuẩn hóa theo batch đầu tiên.
```

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 10]
```diff
@@ -384,7 +391,9 @@ This code is virtually identical to that when we first trained LeNet (:numref:`s
 The main difference is the considerably larger learning rate.
 -->
 
-*dịch đoạn phía trên*
+Như trước đây, ta sẽ huấn luyện trên bộ dữ liệu Fashion-MNIST.
+Đoạn mã này gần như là tương tự với khi chúng ta lần đầu huấn luyên LeNet (: numref: `sec_lenet`).
```
Comment (duythanhvn) [inline chapter_convolutional-modern/batch-norm_vn.md:395]:
```suggestion
Đoạn mã này gần như là tương tự với khi chúng ta lần đầu huấn luyên LeNet (:numref:`sec_lenet`).
```
Chỗ này là code kết nối các phần nội dung, cô gái lưu ý giữ nguyên nhé.

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 11]
```diff
@@ -314,7 +314,13 @@ Also note that for the sake of convenience we did not worry about automatically
 Do not worry, the Gluon `BatchNorm` layer will care of this for us.
 -->
 
-*dịch đoạn phía trên*
+Ta tạm để các chi tiết thuật toán sang một bên mà tập trung vào các khuôn mẫu thiết kế nền tảng cho việc lập trình. 
+Thông thường, ta định nghĩa phần toán trong một hàm riêng biệt, chẳng hạn như `batch_norm`.
+Sau đó, ta tích hợp chức năng này vào một tầng tùy chỉnh, với mã nguồn chủ yếu để giải quyết các vấn đề sổ sách chứng từ,
```
Comment (ngcthuong) [inline chapter_convolutional-modern/batch-norm_vn.md:319]:
```suggestion
Sau đó, ta tích hợp chức năng này vào một tầng tùy chỉnh, với mã nguồn chủ yếu để giải quyết các vấn đề sổ sách chứng từ,
```
Cụm "bookkepping matters" dịch như hiện tại nghe hơi buồn cười. Nên viết theo kiểu là để "dễ tra cứu lại vì bookmark là để mình dò lại dễ.  Như vậy dịch là liên quan tới "cấp phát và quản lý (luồng) dữ liệu" hoặc tương tự.

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 12]
```diff
@@ -314,7 +314,13 @@ Also note that for the sake of convenience we did not worry about automatically
...
+chẳng hạn như di chuyển dữ liệu đến thiết bị phù hợp ngữ cảnh, phân bổ và khởi tạo bất kỳ biến nào được yêu cầu, theo dõi các giá trị trung bình động (của trung bình và phương sai trong trường hợp này), v.v.
```
Comment (ngcthuong) [inline chapter_convolutional-modern/batch-norm_vn.md:320]:
```suggestion
chẳng hạn như di chuyển dữ liệu đến thiết bị phù hợp ngữ cảnh, cấp phát và khởi tạo bất kỳ biến nào được yêu cầu, theo dõi các giá trị trung bình động (của trung bình và phương sai trong trường hợp này), v.v.
```
Allocating --> cấp phát. 

Chỗ này là ám chỉ chuyển dữ liệu đến CPU hoặc GPU tùy điều kiện thiết bị. Dịch theo kiểu hiện tại nghe chưa rõ nghĩa.

File: chapter_convolutional-modern/batch-norm_vn.md [hunk 13]
```diff
@@ -314,7 +314,13 @@ Also note that for the sake of convenience we did not worry about automatically
...
+Khuôn mẫu này cho phép ta tách hoàn toàn các phép tính toán ra khỏi đoạn mã phụ trợ.
```
Comment (ngcthuong) [inline chapter_convolutional-modern/batch-norm_vn.md:321]:
```suggestion
Khuôn mẫu này cho phép ta tách hoàn toàn các phép tính toán ra khỏi đoạn mã phụ trợ.
```
Ở đây có thể dùng là "phần toán" thì sát nghĩa với ý gốc hơn?

File: chapter_convolutional-modern/batch-norm_vn.md (hunk 11 above)
Comment (lkhphuc) [reply chapter_convolutional-modern/batch-norm_vn.md:319]:
```suggestion
Sau đó, ta tích hợp chức năng này vào một tầng tùy chỉnh, với mã nguồn chủ yếu để giải quyết các vấn đề quản trị,
```

Comment (lkhphuc) [reply chapter_convolutional-modern/batch-norm_vn.md:319]:
```suggestion
Sau đó, ta tích hợp chức năng này vào một tầng tùy chỉnh, với mã nguồn chủ yếu để giải quyết các vấn đề quản trị,
```
